vulyk/tasks/__init__.py

```diff
@@ -13,17 +14,18 @@ def init_tasks(app):
     :rtype: dict
     """
     task_types = {}
-
-    for tt in app.config.get("TASK_TYPES", []):
+    enabled_tasks = app.config.get("ENABLED_TASKS", {})
+    for plugin, task in enabled_tasks.iteritems():
         settings = {
             "redundancy": app.config["USERS_PER_TASK"]
         }
-
-        if isinstance(tt, dict):
-            tt = tt["task"]
-            settings.update(tt.get("settings", {}))
-
-        task_type = import_string(tt)(**settings)
-        task_types[task_type.type_name] = task_type
+        task_settings = import_string(
+            "{plugin_name}.settings".format(plugin_name=plugin)
+        )
+        for settings_key in dir(task_settings):
```

Comment:
What for? I mean, why should it have an access to framework settings at all?
What if one will have 2 or 3 plugins (I definitely will).